docs/changelog.md

```diff
@@ -1,6 +1,11 @@
 _See [here](./versioning-and-upgrading.md) for information about versioning and upgrading_
 
 # TBD
+### Changes
+* Imports datastore service and api service clients to interact with both services. 
+* Checks for services availability using two possibles ways: 1) a method from the client and in some tests and 2) the method `WaitForEndpointAvailability` from `NetworkContext` in others.
```

Comment:
Recommend: "1) in some tests, a method from the API/datastore `example-microservices` client and 2) in others, the method `WaitForEndpointAvailability` from `NetworkContext`"